src/chapters/setup-intro-data-types/naming-conventions.rst

```diff
@@ -0,0 +1,53 @@
+.. _naming-conventions:
+
+Java Naming conventions
+=======================
+
+Java has some very straightforward naming conventions. These are
+universally used by Java programmers, and differ in some cases from
+conventions commonly used in other languages.
+
+Again, these are conventions. Not following them will not prevent your
+code from running, as long as you are following Java’s `naming
+rules <http://docs.oracle.com/javase/specs/jls/se8/html/jls-3.html#jls-3.8>`__.
+Java’s identifier naming rules are somewhat hard to parse, so a good
```

Comment:
Should this go in a new paragraph?

Reply:
yes! i re-worded this based on your other comment from this section